.editorconfig

```diff
@@ -0,0 +1,264 @@
+; EditorConfig to support per-solution formatting.
```

Comment:
Is this the standard editor config when creating a new project?

Reply:
https://github.com/dotnet/aspnetcore/blob/main/.editorconfig

Reply:
It's the editorconfig I found on the offcial aspnet core repository (https://github.com/dotnet/aspnetcore).